crates/ott-balancer/src/selection.rs

```diff
@@ -0,0 +1,15 @@
+use crate::monolith::BalancerMonolith;
+
+#[derive(Debug, Default)]
+pub struct MonolithRegistry;
```

Comment:
nit: `MonolithRegistry` isn't a great name. It should describe the strategy it uses to balance load, which is by minimizing the rooms on each Monolith. So something like `MinRoomsSelector`.